application/src/main/java/org/togetherjava/tjbot/features/chaptgpt/ChatGPTService.java

```diff
@@ -0,0 +1,69 @@
+package org.togetherjava.tjbot.features.chaptgpt;
+
+import com.theokanning.openai.OpenAiHttpException;
+import com.theokanning.openai.completion.chat.ChatCompletionRequest;
+import com.theokanning.openai.completion.chat.ChatMessage;
+import com.theokanning.openai.completion.chat.ChatMessageRole;
+import com.theokanning.openai.service.OpenAiService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Service used to communicate to OpenAI API to generate responses.
+ */
+public class ChatGPTService {
+    private ChatGPTService() {
+        throw new UnsupportedOperationException("Utility class, construction not supported");
+    }
+
+    private static final Logger logger = LoggerFactory.getLogger(ChatGPTService.class);
+    private static final OpenAiService openAiService;
+    private static final long TIMEOUT_DURATION = 10000L; // In milliseconds
+    private static final int MAX_TOKENS = 3000;
+    private static final String TOKEN = System.getenv("OPENAI_TOKEN");
```

Comment:
u need to get it from the config instead, read wiki

Reply:
So I have done this but I think a Moderator (or someone higher up) has to access the on the VPS. Further, my changes made it so that `ChatGPTService` is no longer strictly static. Unfortunate since I was hoping that other classes would be able to more easily access this functionality. I suppose I could just create a Singleton.